Add 'Building Source' to docs (#100)
files-community/files-community.github.io: docs/articles/building-source.md

# Building Files from source

1. Install Microsoft Visual Studio and the UWP Development kit.
2. Clone the [repository](https://github.com/files-community/files) and open `Files.sln` in VS.
3. Make sure that `Files.Package` is set as the startup item by right-clicking on `Files.Package` in the solution explorer and hitting 'Set as Startup item'.

> 4. Run `Files.Package, x64`* as `Debug` and it should look like this:

Author: ```suggestion
4. In the architecture pane, select `Files.Package, x64`* as `Debug` which should look like this:
```
I don't want it to be too vague so is this good?

# Building Files from source

> 1. Install Microsoft Visual Studio and the UWP Development kit.

Reviewer: Any version of Visual Studio?

Author: Wdym?

Reviewer: Should it be 2019 and above? Or is VS 6 fine too (I doubt that but you get the idea)

Author: yes, should be VS2019

Author: and above

Author: nooo, yair just said it was 17 and above
```suggestion
1. Install Microsoft Visual Studio 2017 or later and the UWP Development kit.
```

2. Clone the [repository](https://github.com/files-community/files) and open `Files.sln` in VS

> 3. Make sure that `Files.Package` is set as the startup item

Reviewer 2: Can you provide instructions on how someone can set the correct startup project?

Author: what do you mean?

Reviewer 2: Step 3 is to set Files.Package as the startup item, can you provide instructions on how to do that?

Author: Done!

> 4. Run `Files.Package`, x64, Release and it should look like this:

Reviewer 2: ```suggestion
4. Run `Files.Package`, x64, Debug and it should look like this:
```

Author: For me debug doesn't work

> 4. Run `Files.Package, x64` as `Debug` and it should look like this:

Reviewer: Should be generic as in "Choose the architecture for your system"

Author: How about now?
`Install Microsoft Visual Studio and the UWP Development kit.
Clone the repository and open Files.sln in VS.
Make sure that Files.Package is set as the startup item by right-clicking on Files.Package in the solution explorer and hitting 'Set as Startup item'.
Run Files.Package, x64* as Debug and it should look like this: VS Run config image
You're good to go!
If x64 doesn't work, use the architecture of your system`

Author: How about now?
![image](https://user-images.githubusercontent.com/71598437/132376707-bc85ec8b-a265-4b68-aed2-2ad44dad625a.png)

> ![VS Run config image](https://raw.githubusercontent.com/DeveloperWOW64/files-community.github.io/main/img/vs_run_cfg.png)

Reviewer 2: Can you include this image in the PR?

Author: wdym?

Reviewer: ```suggestion
![VS Run config image](../../img/vs_run_cfg.png)
```

I think he means this.
